libs/openchallenges/challenge-search/src/lib/challenge-search-filters.ts

```diff
@@ -1,88 +1,163 @@
-};
+export const challengeStatusFilter: Filter[] = [
+  {
+    value: 'active',
+    label: 'Active',
+  },
+  {
+    value: 'upcoming',
+    label: 'Upcoming',
+  },
+  {
+    value: 'completed',
+    label: 'Completed',
+  },
+];
 
-export const challengeIncentivesFilter: Filter = {
-  query: 'incentives',
-  label: 'Incentive Type',
-  values: challengeIncentivesFilterValues,
-  collapsed: false,
-};
+export const challengeDifficultyFilter: Filter[] = [
+  {
+    value: 'good_for_beginners',
+    label: 'Good For Beginners',
+  },
+  {
+    value: 'intermediate',
+    label: 'Intermediate',
+  },
+  {
+    value: 'advanced',
+    label: 'Advanced',
+  },
+];
 
-export const challengePlatformsFilter: Filter = {
-  query: 'platforms',
-  label: 'Platform',
-  values: challengePlatformsFilterValues,
-  collapsed: false,
-};
+export const challengeSubmissionTypesFilter: Filter[] = [
+  {
+    value: 'container_image',
+    label: 'Container Image',
+  },
+  {
+    value: 'prediction_file',
+    label: 'Prediction File',
+  },
+  {
+    value: 'notebook',
+    label: 'Notebook',
+  },
+  {
+    value: 'other',
+    label: 'Other',
+  },
+];
 
-// dropdown filters
-export const challengeInputDataTypesFilter: Filter = {
-  query: 'inputDataTypes',
-  label: 'Input Data Type',
-  values: challengeInputDataTypesFilterValues,
-  collapsed: false,
-  showAvatar: false,
-};
+export const challengeIncentivesFilter: Filter[] = [
+  {
+    value: 'monetary',
+    label: 'Monetary',
+  },
+  {
+    value: 'publication',
+    label: 'Publication',
+  },
+  {
+    value: 'speaking_engagement',
+    label: 'Speaking Engagement',
+  },
+  {
+    value: 'other',
+    label: 'Other',
+  },
+];
 
-export const challengeCategoriesFilter: Filter = {
-  query: 'categories',
-  label: 'Category',
-  values: challengeCategoriesFilterValues,
-  collapsed: false,
-};
+export const challengePlatformsFilter: Filter[] = [];
 
-export const challengeOrganizationsFilter: Filter = {
-  query: 'organizations',
-  label: 'Organization',
-  values: challengeOrganizationsFilterValues,
-  collapsed: false,
-  showAvatar: true,
-};
+export const challengeInputDataTypesFilter: Filter[] = [];
 
-export const challengeOrganizatersFilter: Filter = {
-  query: 'organizers',
-  label: 'Organizer',
-  values: challengeOrganizersFilterValues,
-  collapsed: false,
-  showAvatar: true,
-};
+export const challengeCategoriesFilter: Filter[] = [
+  {
+    value: 'featured',
+    label: 'Featured',
+  },
+  {
+    value: 'starting_soon',
+    label: 'Starting Soon',
+  },
+  {
+    value: 'ending_soon',
+    label: 'Closing Soon',
+  },
+  {
+    value: 'recently_started',
+    label: 'Recently Launched',
+  },
+  {
+    value: 'recently_ended',
+    label: 'Recently Completed',
+  },
+];
+
+export const challengeOrganizationsFilter: Filter[] = [];
+
+export const challengeOrganizersFilter: Filter[] = [];
+
+export const challengeSortFilter: Filter[] = [
+  {
+    value: 'relevance',
```

Comment:
Would that work if we replace the string value by its enum value? This approach is more robust.

```suggestion
    value: ChallengeSort.Relevance,
```